corehq/apps/hqwebapp/views.py

```diff
@@ -351,6 +351,11 @@ def login(req):
     # this view, and the one below, is overridden because
     # we need to set the base template to use somewhere
     # somewhere that the login page can access it.
+
+    if settings.SERVER_ENVIRONMENT == 'icds':
```

Comment:
pretty sure you can make this use `settings.ENTERPRISE_MODE` for this. Currently ICDS is the only env with that set True but I imagine we want the same thing for future enterprise clusters.